pages/HowToPlay.jsx

```diff
@@ -38,26 +34,21 @@ class HowToPlay extends React.Component {
-                        </div>
                     </div>
                     <div className='htp-topic'>
                         <h3 className='htp-title' id='bugs'>Bugs and automation</h3>
-                        <p>Doomtown Online is still a work in progress. While many cards are implemented and should be working correctly, there are quite a few still to be implemented. We keep a list up to date with unimplemented and partially implemented cards <a target='_blank' href='https://github.com/townteki/townsquare/blob/master/docs/cardpool-status.md'>here</a>. If you happen upon a card that you believe is not working as it should and it is marked complete that list, it would help immensely if you would submit an issue on <a target='_blank' href='https://gitreports.com/issue/townteki/townsquare'>GitHub</a>. Other comments and/or feedback can be left on GitHub as well.</p>
+                        <p>Doomtown Online is still a work in progress. While many cards are implemented and should be working correctly, there are quite a few still to be implemented. We keep a list up to date with unimplemented and partially implemented cards <a target='_blank' href='https://github.com/townteki/townsquare/blob/master/docs/cardpool-status.md'>here</a>. If you happen upon a card that you believe is not working as it should and it is marked complete on that list, it would help immensely if you would submit an issue on <a target='_blank' href='https://gitreports.com/issue/townteki/townsquare'>GitHub</a>. Other comments and/or feedback can be left on GitHub as well.</p>
                     </div>
                     <div className='htp-topic'>
                         <h3 className='htp-title' id='interactions'>Specific Card Interactions</h3>
-
+                        <div className='htp-subtopic'>
+                            <h4 className='htp-subtitle' id='marges'>Old Marge's Manor</h4>
+                            <p><a href='https://dtdb.co/en/card/12012' target='blank'>Old Marge's Manor's</a> React ability must be triggered as a normal Noon action instead of a React. When you want to use it, click the card and choose "Use Ability" on the card menu</p>
+                        </div>
+                        <div className='htp-subtopic'>
+                            <h4 className='htp-subtitle' id='johnnybrocklehurst'>Johnny Brocklehurst</h4>
+                            <p><a href='https://dtdb.co/en/card/19017' target='blank'>Johnny Brocklehurst</a>'s React ability must be triggered as a normal Noon action instead of a React. When you want to use it, click the card and choose "Use Ability" on the card menu</p>
+                        </div>
```

Comment:
Let's make this as one section
```suggestion
                            <h4 className='htp-subtitle' id='margesandjohnny'>Old Marge's Manor and Johnny Brocklehurst</h4>
                            <p><a href='https://dtdb.co/en/card/12012' target='blank'>Old Marge's Manor</a>'s and <a href='https://dtdb.co/en/card/19017' target='blank'>Johnny Brocklehurst</a>'s React ability must be triggered as a normal Noon action instead of a React. When you want to use it, click the card and choose "Use Ability" on the card menu</p>
                        </div>
```